pathology/tumor_detection/torch/camelyon_train_evaluate_pytorch_gpu.py

```diff
@@ -518,7 +518,7 @@ def parse_arguments():
     parser.add_argument("--amp", action="store_true", help="activate amp")
     parser.add_argument("--novograd", action="store_true", help="activate novograd optimizer")
     parser.add_argument("--cos", action="store_true", help="activate cosine annealing")
-    parser.add_argument("--pretrain", action="store_true", help="activate Imagenet weights")
+    parser.add_argument("--no-pretrain", action="store_false", dest="pretrain", help="deactivate Imagenet weights")
```

Comment:
Hi @Nic-Ma, this is means to disable pretrained. The reason to have it like to make the pipeline to use pretrained weight by default when nothing is provided (which is almost always).

Reply:
Hi @drbeh, if like that, I will also update the other two files.
https://github.com/Project-MONAI/tutorials/blob/96a38175df51ecee1dc4fbb3499cf3eb6967add7/pathology/tumor_detection/ignite/camelyon_train_evaluate.py#L270
https://github.com/Project-MONAI/tutorials/blob/96a38175df51ecee1dc4fbb3499cf3eb6967add7/pathology/tumor_detection/ignite/camelyon_train_evaluate.py#L183
What do you think?

Reply:
Hi @Nic-Ma, I agree with you. If it needs to be changed the dest should be changed to "disable_pretrain".

Reply:
Hi @KumoLiu, to me actually there is nothing wrong with either way but it is about trade-off between code readability and user convenience. Changing to `--pretrain` flag (instead of `--no-pretrain`) makes the code more readable but the user need to mention it most of the time. I agree to have all the code in the same way. Thanks

Reply:
Do you mean that you want it to be `--no-pretrain`? Both are ok to me.

Reply:
@KumoLiu I think if we are changing `--no-pretrain` to `--pretrain`, we should do the same to others flag, like `--no-novograd`, etc. What do you think?

Reply:
Maybe not? I think you already have different ways to do that.
https://github.com/Project-MONAI/tutorials/blob/24bce8adb2591d2a1326efb1b0eecabc2af93f2b/pathology/tumor_detection/ignite/camelyon_train_evaluate.py#L267-L270

Reply:
And also for this `pretrain` argument, I think users should mention it most of the time since it may be related to the terms of use.